marklogic/api_client.py

```diff
@@ -122,7 +122,13 @@ def get_judgment_xml(self, uri: str) -> str:
     def get_judgments_index(self, page: str) -> requests.Response:
         start = (int(page) - 1) * RESULTS_PER_PAGE + 1
         headers = {"Accept": "multipart/mixed"}
-        return self.GET("LATEST/search/?view=results&start=" + str(start), headers)
+        return self.GET(
+            "LATEST/search/?view=results&start="
+            + str(start)
+            + "&pageLength="
+            + str(RESULTS_PER_PAGE),
```

Comment:
@lozette Python's `f-string` syntax might be a little clearer here. You could rewrite this string as:
```suggestion
            f"LATEST/search/?view=results&start={start}&pageLength={RESULTS_PER_PAGE}",
```